shcDecoder/src/main/java/ca/bc/gov/shcdecoder/cache/impl/FileManagerImpl.kt

```diff
@@ -6,16 +6,20 @@ import ca.bc.gov.shcdecoder.cache.FileManager
 import ca.bc.gov.shcdecoder.model.Issuer
 import ca.bc.gov.shcdecoder.model.Jwks
 import ca.bc.gov.shcdecoder.model.JwksKey
+import ca.bc.gov.shcdecoder.model.RevocationsResponse
 import ca.bc.gov.shcdecoder.model.Rule
 import ca.bc.gov.shcdecoder.model.TrustedIssuersResponse
 import ca.bc.gov.shcdecoder.model.ValidationRuleResponse
+import ca.bc.gov.shcdecoder.utils.epochToDate
 import com.google.gson.Gson
 import java.io.File
 import java.io.FileOutputStream
 import java.net.URL
+import java.util.Date
 
 class FileManagerImpl(
-    context: Context
+    context: Context,
+    private val gson: Gson
```

Comment:
Please revert this variable declaration to method scope.
with refactored code, I don't see its usage beyond one function.